rules/do-you-display-dates-in-a-human-friendly-format/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 archivedreason: 
-title: Do you display dates in a human friendly format?
```

Comment:
I think this title makes sense.

Reply:
Changed to **Do you display completed time and date in a human friendly format?**